chainstate/src/config.rs

```diff
@@ -31,7 +31,12 @@ make_config_setting!(
     )
 );
 make_config_setting!(TxIndexEnabled, bool, false);
-make_config_setting!(MaxTipAge, Duration, Duration::from_secs(60 * 60 * 24));
+make_config_setting!(
+    MaxTipAge,
+    Duration,
+    // TODO: temporary fix so that functional tests still work
```

Comment:
If this is meant to be reverted before the PR is merged, please add TODO(PR). Otherwise, please explain in more detail what this fix does.

Reply:
Updated to `TODO(PR)`.